org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/interpreter/DictionaryTestCase.xtend

```diff
@@ -0,0 +1,231 @@
+package org.uqbar.project.wollok.tests.interpreter
+
+import org.junit.Test
+import org.uqbar.project.wollok.interpreter.core.WollokProgramExceptionWrapper
+
+/**
+ * @author jfernandes
+ */
+class DictionaryTestCase extends AbstractWollokInterpreterTestCase {
+	
+	def instantiateObjectsDictionary() {
+		"val dictionary = {foo:1, bar:'hello', baz: {:}}"
```

Comment:
```{ : }``` seems weird.
What would be that for ?
Can we just forbid that at grammar level ?